build/Makefile

```diff
@@ -392,8 +392,6 @@ gen-crd-client: $(ensure-build-image)
 	docker run --rm $(common_mounts) -w $(workdir_path) $(build_tag) /root/gen-crd-client.sh
 	docker run --rm $(common_mounts) -w $(workdir_path)/pkg $(build_tag) goimports -w .
 
-site-images: $(site_path)/static/diagrams/gameserver-states.dot.png $(site_path)/static/diagrams/gameserver-lifecycle.puml.png
-
 # generate pngs from dot files
```

Comment:
Should this part of the makefile move as well? Are there places outside of the website where we generate pngs from dot files and puml files?

Reply:
That is true that we don't - but it seemed general enough that maybe leave it where it was.

Honestly, I've no strong opinion on this, so if you feel strongly, I'll move it.

Reply:
If we are still thinking about moving the website to a different repo after 1.0 I'd move it to the site makefile now to make that simpler.

Reply:
Makes perfect sense. Will make the move :+1: